queue/driver_mongo.go

```diff
@@ -111,7 +109,6 @@ func DefaultMongoDBOptions() MongoDBOptions {
 	return MongoDBOptions{
```

Comment:
Can't comment on the line, but remove the mention of priority in the doc comment.

Reply:
fixed. And I realized my changes in mock didn't save, fixed it now.